README.md

```diff
@@ -1,15 +1,62 @@
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+<a href="mailto:sales@syncfusion.com?Subject=Syncfusion Blazor Split Buttons - GitHub" target="_top">sales@syncfusion.com</a> | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples">www.syncfusion.com</a>| Toll Free: 1-888-9 DOTNET <br>
```

Comment:
![image](https://user-images.githubusercontent.com/29474815/178662888-cf45b76c-7be8-494b-acba-016c40727a53.png)
Remove the `s`

Reply:
Removed